### ISSUE TITLE
wrong in text color 

### ISSUE THREAD

**Issue by i2863CookieZJ** [wrapp/floatlabelededittext#50]
i create the fletTextAppearance in style,and set textcolor,when the edittext get focus,flettext's color is right,but the textcolor is not work when lose focus.why? how to solve ?

**Comment by LunevNF** [comment]
Yeah, I have the same issue. What's wrong? How to set color of label for not focused edittext?

**Comment by LunevNF** [comment]
Ok. I was try to set color selector (res/color/label_color_selector.xml):

    <selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:state_focused="true" android:color="@color/colorLabeledEditText"/>
    <item android:state_pressed="true" android:state_enabled="false" android:color="@color/colorLabeledEditText" />
    <item android:state_enabled="false" android:color="@color/colorLabeledEditText" />
    <item android:color="@color/colorLabeledEditText"/>
</selector>

res/values/styles.xml:

    <style name="floatlabelededittext">
        <item name="android:textColor">@color/label_color_selector</item>
    </style>

But it steel not working!

**Comment by LunevNF** [comment]
Ok. Just install rengwuxian/MaterialEditText and all be OK. This author don't want to help.

**Comment by heinrisch** [comment]
@nikitoSha Float labels are part of the support library now. I don't see why you would use a third party implementation...?

**Comment by LunevNF** [comment]
А тебя это колышет что ли? По факту: у тебя есть библиотека, она херово отрабатывает. Решить проблему ты не можешь. Поэтому, естественно, я сейчас использую стандартный набор.
